src/taxonomies/index.js

```diff
@@ -37,6 +37,27 @@ export const getTermBySlug = async ( requestUtils, taxonomy, slug ) => {
 	} );
 };
 
+/**
+ * Creates a term.
+ *
+ * @param {RequestUtils} requestUtils       Gutenberg request utils object.
+ * @param {string}       taxonomy           Taxonomy slug (e.g., 'categories', 'tags').
```

Comment:
```suggestion
 * @param {string}       taxonomy           Taxonomy slug used in related REST endpoint (e.g., 'categories', 'tags').
```
Proposing a more precise doc.